pages/index.js

```diff
@@ -3,16 +3,27 @@ import React from "react";
 import FindMultisigForm from "../components/forms/FindMultisigForm";
 import Page from "../components/layout/Page";
 import StackableContainer from "../components/layout/StackableContainer";
+import { useAppContext } from "../context/AppContext";
 
-const MultiPage = () => (
-  <Page>
-    <StackableContainer base>
-      <StackableContainer lessPadding>
-        <h1 className="title">Cosmoshub Multisig Manager</h1>
+const MultiPage = () => {
+  const { state } = useAppContext();
+  return (
+    <Page>
+      <StackableContainer base>
+        <StackableContainer lessPadding>
+          <h1 className="title">
+            <span>{state.chain.chainDisplayName}</span> Multisig Manager
+          </h1>
+        </StackableContainer>
+        <FindMultisigForm />
       </StackableContainer>
-      <FindMultisigForm />
-    </StackableContainer>
-  </Page>
-);
+      <style jsx>{`
+        span {
+          text-transform: capitalize;
```

Comment:
I think this capitalization should be done somewhere at the data level. E.g. "cosmoshub" should be called "Cosmos Hub".

Can we just use the [pretty_name](https://github.com/cosmos/chain-registry/blob/master/cosmoshub/chain.json#L5 ) instead and print it unmodified?

Reply:
Ya this makes sense, I think i overlooked the `pretty_name` field 😅

fixed in
3155a53647e137cc722464e65db41202ac0f971d and
6e56587a9ed61c76a5c1c50ef27c133666d9a7b0